controlplane/api/connection/connection_helpers.go

```diff
@@ -28,16 +28,16 @@ func (c *Connection) IsRemote() bool {
 		return false
 	}
 	// If we have two or more, it is remote
-	return len(c.NetworkServiceManagers) > 1
+	return len(c.GetPath().GetPathSegments()) > 1
```

Comment:
It is not right with segments, since if will be more segments because of forwarder.